canonical-web-and-design/ubuntu.com
# Add sitemap index

## Done

- Add sitemap index
- Remove erroring links from sitemap

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/8864

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/sitemap.xml
- Should get a xml rendered page containing the links to the different sitemaps of the site
